src/glue.d

```diff
@@ -970,6 +968,15 @@ void FuncDeclaration_toObjFile(FuncDeclaration fd, bool multiobj)
             objmod.includelib(libname);
             s.Sclass = SCglobal;
         }
+        else if (strcmp(s.Sident.ptr, "rt_init") == 0 && fd.linkage == LINKc)
```

Comment:
I thought so, too, but just copied the condition checking for C "main". Should this be changed, too?